action.yaml

```diff
@@ -7,6 +7,9 @@ inputs:
   secret:
     description: Secret
     required: true
+  minimum_approvals:
```

Comment:
Is this format (underscore separating words) the accepted and common standard for actions inputs? Can you link an example of another action that uses this format? Not saying it's wrong, just want to make sure that we're following any standard that might exist.

Reply:
This is a fair comment - in actions created by Github, they prefer hyphen separated inputs as a standard. Happy to change this.